metatensor-torch/src/misc.cpp

```diff
@@ -54,3 +54,20 @@ void metatensor_torch::save(const std::string& path, TorchTensorMap tensor) {
 torch::Tensor metatensor_torch::save_buffer(TorchTensorMap tensor) {
     return tensor->save_buffer();
 }
+
+
+TorchLabels metatensor_torch::load_labels(const std::string& path) {
+    return LabelsHolder::load(path);
+}
+
+TorchLabels metatensor_torch::load_labels_buffer(torch::Tensor buffer) {
+    return LabelsHolder::load_buffer(buffer);
+}
+
+void metatensor_torch::save(const std::string& path, TorchLabels labels) {
```

Comment:
How about `save_labels` since this can only save labels?
Also, later we use a `load` function to load `TensorMap`s

Reply:
So I had a discussion here with @PicoCentauri who suggested this approach. We can make the save function take both labels and tensormap, and it does make for a cleaner API I think.

This is harder to do for loading, since overloading on return type is not possible in C++.

I'm not opposed to making the functions more symetric and have this function renamed to `save_labels` everywhere.

Reply:
I missed the overload. I think it's a good solution